docker/docker-compose-aspnetcore/redis/oats.redis-lpush.yml

```diff
@@ -0,0 +1,14 @@
+include:
```

Comment:
For me this file needs to be end in `.yaml` rather than in `.yml`, otherwise it isn't picked up by OATS' automatic test
 detection.